timemachine/fe/atom_mapping.py

```diff
@@ -437,9 +417,8 @@ def leaf_filter_fxn(trial_core) -> bool:
         make_leaf_filter_fxn(),
     )
 
-    all_bond_cores = [_compute_bond_cores(mol_a, mol_b, marcs) for marcs in all_marcs]
     all_cores = remove_cores_smaller_than_largest(all_cores)
-    all_cores, _ = _deduplicate_all_cores_and_bonds(all_cores, all_bond_cores)
```

Comment:
While this returned the right cores, the bonds could be meaningless because we had removed all of the smaller cores.

Changed to only de-duplicate on cores since we don't use the bonds for post-processing any longer.